packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -448,21 +446,36 @@ export const PSM_MANIFEST = harden({
       chainTimerService: 'timer',
       psmFacets: true,
     },
+    produce: { psmFacets: 'true' },
     installation: {
       consume: { contractGovernor: 'zoe', psm: 'zoe' },
     },
     instance: {
       consume: { economicCommittee: 'economicCommittee' },
     },
     brand: {
-      consume: { AUSD: 'bank', IST: 'zoe' },
-    },
-    issuer: {
-      consume: { AUSD: 'bank' },
+      consume: { [Stable.symbol]: 'zoe' },
     },
   },
 });
 
+export const getManifestForPsmGovernance = (
+  { restoreRef },
+  { installKeys },
+) => {
+  const { [installGovAndPSMContracts.name]: _toss, ...manifest } =
```

Comment:
isn't simply `_` the convention here for ignoring a param?
```suggestion
  const { [installGovAndPSMContracts.name]: _, ...manifest } =
```